src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -177,6 +177,11 @@ public enum NuGetLogCode
         /// </summary>
         NU1109 = 1109,
 
+        /// <summary>
+        /// The package `packageId` is available in the Global packages folder, but the source it came from `package source URI` is not one of this solution's configured sources.
+        /// </summary>
+        NU1110 = 1110,
```

Comment:
I think this error code makes most sense in this category. 
NU1100 includes resolver errors which includes missing packages, missing versions etc. In a way, this is a missing package.

Reply:
Great👍...